Makefile

```diff
@@ -12,42 +12,42 @@
-	v1.11/arm64/debian-elasticsearch6:v1.11.3-debian-elasticsearch6-arm64-1.0,v1.11-debian-elasticsearch6-arm64-1 \
-	v1.11/arm64/debian-loggly:v1.11.3-debian-loggly-arm64-1.0,v1.11-debian-loggly-arm64-1 \
-	v1.11/arm64/debian-logentries:v1.11.3-debian-logentries-arm64-1.0,v1.11-debian-logentries-arm64-1 \
-	v1.11/arm64/debian-cloudwatch:v1.11.3-debian-cloudwatch-arm64-1.0,v1.11-debian-cloudwatch-arm64-1 \
-	v1.11/arm64/debian-stackdriver:v1.11.3-debian-stackdriver-arm64-1.0,v1.11-debian-stackdriver-arm64-1 \
-	v1.11/arm64/debian-s3:v1.11.3-debian-s3-arm64-1.0,v1.11-debian-s3-arm64-1 \
-	v1.11/arm64/debian-syslog:v1.11.3-debian-syslog-arm64-1.0,v1.11-debian-syslog-arm64-1 \
-	v1.11/arm64/debian-forward:v1.11.3-debian-forward-arm64-1.0,v1.11-debian-forward-arm64-1 \
-	v1.11/arm64/debian-gcs:v1.11.3-debian-gcs-arm64-1.0,v1.11-debian-gcs-arm64-1 \
-	v1.11/arm64/debian-graylog:v1.11.3-debian-graylog-arm64-1.0,v1.11-debian-graylog-arm64-1 \
-	v1.11/arm64/debian-papertrail:v1.11.3-debian-papertrail-arm64-1.0,v1.11-debian-papertrail-arm64-1 \
-	v1.11/arm64/debian-logzio:v1.11.3-debian-logzio-arm64-1.0,v1.11-debian-logzio-arm64-1 \
-	v1.11/arm64/debian-kafka:v1.11.3-debian-kafka-arm64-1.0,v1.11-debian-kafka-arm64-2 \
-	v1.11/arm64/debian-kinesis:v1.11.3-debian-kinesis-arm64-2.0,v1.11-debian-kinesis-arm64-2
+	v1.11/arm64/debian-azureblob:v1.11.4-debian-azureblob-arm64-1.0,v1.11-debian-azureblob-arm64-1 \
+	v1.11/arm64/debian-elasticsearch7:v1.11.4-debian-elasticsearch7-arm64-1.0,v1.11-debian-elasticsearch7-arm64-1,v1-debian-elasticsearch-arm64 \
+	v1.11/arm64/debian-elasticsearch6:v1.11.4-debian-elasticsearch6-arm64-1.0,v1.11-debian-elasticsearch6-arm64-1 \
+	v1.11/arm64/debian-loggly:v1.11.4-debian-loggly-arm64-1.0,v1.11-debian-loggly-arm64-1 \
+	v1.11/arm64/debian-logentries:v1.11.4-debian-logentries-arm64-1.0,v1.11-debian-logentries-arm64-1 \
+	v1.11/arm64/debian-cloudwatch:v1.11.4-debian-cloudwatch-arm64-1.0,v1.11-debian-cloudwatch-arm64-1 \
+	v1.11/arm64/debian-stackdriver:v1.11.4-debian-stackdriver-arm64-1.0,v1.11-debian-stackdriver-arm64-1 \
+	v1.11/arm64/debian-s3:v1.11.4-debian-s3-arm64-1.0,v1.11-debian-s3-arm64-1 \
+	v1.11/arm64/debian-syslog:v1.11.4-debian-syslog-arm64-1.0,v1.11-debian-syslog-arm64-1 \
+	v1.11/arm64/debian-forward:v1.11.4-debian-forward-arm64-1.0,v1.11-debian-forward-arm64-1 \
+	v1.11/arm64/debian-gcs:v1.11.4-debian-gcs-arm64-1.0,v1.11-debian-gcs-arm64-1 \
+	v1.11/arm64/debian-graylog:v1.11.4-debian-graylog-arm64-1.0,v1.11-debian-graylog-arm64-1 \
+	v1.11/arm64/debian-papertrail:v1.11.4-debian-papertrail-arm64-1.0,v1.11-debian-papertrail-arm64-1 \
+	v1.11/arm64/debian-logzio:v1.11.4-debian-logzio-arm64-1.0,v1.11-debian-logzio-arm64-1 \
+	v1.11/arm64/debian-kafka:v1.11.4-debian-kafka-arm64-1.0,v1.11-debian-kafka-arm64-2 \
+	v1.11/arm64/debian-kinesis:v1.11.4-debian-kinesis-arm64-2.0,v1.11-debian-kinesis-arm64-2
```

Comment:
Using `-2.0` suffix is reasonable?
I'd bumped fluent-plugin-kinesis plugin version to v3.2.3 in the past few weeks.

Reply:
What is the proper version? `-1.0`?
Could you change the version in this branch?

Reply:
Sure. Will do.